_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,172 @@
+
+There may be more types of roles available depending on the platforms, but these are the main buckets we think of when working in the open.
+
+Depending on the platform you use, various controls may be offered to let you openly share your documents and your work while remaining in control of its content.
+
+In SharePoint, a platform most often available as "Inner Source" only, a team can open up their workspace by default to their colleagues and create specific folders with more restricted access to privately work on documents only when absolutely required.
+
+With most collaboration platforms, whether it's SharePoint, Google Documents, etc., there usually is an option to let external contributors consult your documents and add comments but prevent them from making changes directly.
+This way, you remain in control of the main version but you enable people to stop by and provide feedback on a continuous basis.
+
+### Ensure Privacy
+
+Public servants who work with citizen data must take great care regarding the privacy of that data and never share it publicly.
+Working in the open by default is not a concept that applies when working on a citizen's files and data.
+
+But it doesn't mean that even these teams can't work openly on improving their processes and sharing how they overcomed challenges.
+As public servants, we should continuously strive to improve our processes, look for new ways of solving issues, and share the results of our researches, as long as they are unclassified.
+
+### Manage Secrets
+
+In software development, certain parts of the final code may be a corporate secret, such as a private key to a computer resource, passwords, etc.
+
+These must never be included in the public source code itself but rather be managed separately.
+
+Since, software source code is nothing more than structured text that can be understood by a computer, the same concept of pulling out secrets parts may be taken with other types of documents.
+
+This means that when a small part of your work must not be shared publicly, you can exclude it and store it in a secure location instead of locking down the entire document.
+
+This way, the material that is unclassified can still be made available publicly and may be collectively worked on while the parts that are to be secured can be edited separately only by the select individuals who are authorized.
+
+### Manage Leaks
+
+In software development, we expect that there will be a security issue at some point, while we continue to strive to minimize that eventuality.
+
+By putting in place a remediation process, we ensure that even if there was to be a leak, appropriate mesures are in place to address it as quickly as possible.
+And we make sure that everyone knows how to run through the process, if it's not fully automated.
+
+In the optic of traditional text documents and presentations, reducing the amount of specific data used may help minimize the risks of leaks.
```

Comment:
If keeping this section, I would change 'in the optic of' to simply 'For traditional text....'